src/onepasswordconnectsdk/async_client.py

```diff
@@ -71,7 +71,7 @@ async def get_file_content(self, file_id: str, item_id: str, vault_id: str, cont
         return response.content
 
     async def download_file(self, file_id: str, item_id: str, vault_id: str, path: str):
```

Comment:
Could you add `-> None` here, to stay consistent with the annotation in `client.py`?